ext/opentelemetry-ext-http-requests/src/opentelemetry/ext/http_requests/__init__.py

```diff
@@ -50,8 +51,8 @@ def enable(tracer):
 
     @functools.wraps(wrapped)
     def instrumented_request(self, method, url, *args, **kwargs):
-        # TODO: Check if we are in an exporter, cf. OpenCensus
-        # execution_context.is_exporter()
+        if Context.is_exporter:  # Check if we are in an exporter
```

Comment:
Maybe we can generalize this concept into a "terminal span" as @a-feld mentioned. Save this for the post alpha v0.1 work.

Reply:
Changed to `suppress_instrumentation` as suggested by @Oberon00.